analysis/tweet_entities_analysis/get_tweet_entities.py

```diff
@@ -0,0 +1,160 @@
+import json
+import csv
+import requests
+import regex
+import os
+from datetime import datetime
+
+TWEETS_SOURCE_FOLDER = '../formated_data/tweet/'
+POLITICIANS_LIST = '../../assets/all_politicians.json'
+
+def get_politicians_list():
+    politicians = []
+    with open(POLITICIANS_LIST, 'r', encoding='utf-8') as infile:
+        for p in json.load(infile):
+            politicians.append(p)
+    return politicians
+
+politicians_list = get_politicians_list()
```

Comment:
Could just be

    with open(POLITICIANS_LIST, 'r', encoding='utf-8') as infile:
          politicians_list = [p for p in json.load(infile)]